R/verify_input.r

```diff
@@ -56,23 +55,14 @@ verify_input <- function(model_dir, type = c("om", "em")) {
       f.dat <- NA
     }
   }
-  if (type == "om") {
 
-    # commented out, because the inital .par will be generated in ss3sim_base(),
-    # to ensure it is consistent with the ctl file.
-    # if (length(grep("ss.par", files, ignore.case = TRUE))) {
-    #   f.par <- grep("ss.par", files, ignore.case = TRUE)
-    # } else {
-    #   f.par <- NA
-    # }
-  }
   if (length(grep("starter.ss", files, ignore.case = TRUE))) {
-    f.starter <- grep("starter.ss", files, ignore.case = TRUE)
+    f.starter <- grep("starter.ss$", files, ignore.case = TRUE)
   } else {
     f.starter <- NA
   }
   if (length(grep("forecast.ss", files, ignore.case = TRUE))) {
-    f.forecast <- grep("forecast.ss", files, ignore.case = TRUE)
+    f.forecast <- grep("forecast.ss$", files, ignore.case = TRUE)
```

Comment:
Tip: you could also add ^ in front to fix it as the start of the string.

Reply:
Thanks, changed in 9c32e68430f2b7c9e00c076a3a77f3d57d80153f.